dist/scss/core/mixins/_grid-mixins.scss

```diff
@@ -1,11 +1,12 @@
+@use 'sass:math';
 /// Grid system
 //
 // Generate semantic grid columns with these mixins.
 
 @mixin grid-container() {
   width: 100%;
-  padding-right: ($grid-gutter-width / 2);
-  padding-left: ($grid-gutter-width / 2);
+  padding-right: math.div($grid-gutter-width, -2);
+  padding-left: math.div($grid-gutter-width, -2);
```

Comment:
As I read the documentation, you should divide by `2` instead of `-2`, or am I missing something?

https://sass-lang.com/documentation/breaking-changes/slash-div

Reply:
you are absolutely right, typo from my part, fixed in latest commit